CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+x.y.z Release notes (yyyy-MM-dd)
+=============================================================
+### Enhancements
+* Add ability to save a NotificationToken when using Combine for observing an Object or Collection by using `saveToken(on:for:)` before `sink` in a chain.
```

Comment:
Using this doesn't require that `.sink` appears in the chain and the position is quite a bit more constrained than that.